components/scream/src/physics/p3/tests/p3_unit_tests.cpp

```diff
@@ -451,6 +451,186 @@ static void  cloud_water_autoconversion_unit_bfb_tests(){
+	 -6.2915E+07},
+
+	{1.9746E-08, 1.8779E+07, 1.2238E-11, 1.1639E+04, 0.0000E+00, 7.3471E+03, 0.0000E+00, 0.0000E+00, 0.0000E+00, 6.1031E+04,
+	 true      , 9.4705E-01, 1.0531E+00, 2.5010E+06, 1.8000E+03, 2.8958E+02, 5.0000E-03, 3.5565E-05,-6.8117E+09, 1.1308E-04,
+	 -7.4813E+07},
+
+	{2.3047E-08, 1.8779E+07, 1.4521E-11, 1.1832E+04, 0.0000E+00, 7.4688E+03, 0.0000E+00, 0.0000E+00, 0.0000E+00, 6.0474E+04,
+	 true      , 9.5579E-01, 1.0582E+00, 2.5010E+06, 1.8000E+03, 2.8941E+02, 4.7949E-03, 4.1510E-05,-6.7792E+09, 1.4787E-05,
+	 -8.2885E+07},
+
+	{2.6289E-08, 1.8779E+07, 1.6845E-11, 1.2033E+04, 0.0000E+00, 7.5955E+03, 0.0000E+00, 0.0000E+00, 0.0000E+00, 5.9907E+04,
+	 true      , 9.6483E-01, 1.0634E+00, 2.5010E+06, 1.8000E+03, 2.8972E+02, 4.4341E-03, 4.7350E-05,-6.7452E+09,-4.7350E-05,
+	 -8.3634E+07},
+
+	{2.9533E-08, 1.8779E+07, 1.9253E-11, 1.2242E+04, 0.0000E+00, 7.7277E+03, 0.0000E+00, 0.0000E+00, 0.0000E+00, 5.9332E+04,
+	 true      , 9.7418E-01, 1.0686E+00, 2.5010E+06, 1.8000E+03, 2.9002E+02, 4.0751E-03, 5.3194E-05,-6.7096E+09,-5.3194E-05,
+	 -8.4862E+07},
+
+	{3.2784E-08, 1.8780E+07, 2.1753E-11, 1.2461E+04, 0.0000E+00, 7.8657E+03, 0.0000E+00, 0.0000E+00, 0.0000E+00, 5.8748E+04,
+	 true      , 9.8387E-01, 1.0741E+00, 2.5010E+06, 1.8000E+03, 2.9033E+02, 3.7211E-03, 5.9050E-05,-6.6723E+09,-5.9050E-05,
+	 -8.6159E+07},
+
+	{3.6045E-08, 1.8780E+07, 2.4356E-11, 1.2689E+04, 0.0000E+00, 8.0098E+03, 0.0000E+00, 0.0000E+00, 0.0000E+00, 5.8154E+04,
+	 true      , 9.9391E-01, 1.0796E+00, 2.5010E+06, 1.8000E+03, 2.9063E+02, 3.3756E-03, 6.4925E-05,-6.6333E+09,-6.4925E-05,
+	 -8.7530E+07},
+
+	{3.9321E-08, 1.8780E+07, 2.7069E-11, 1.2928E+04, 0.0000E+00, 8.1605E+03, 0.0000E+00, 0.0000E+00, 0.0000E+00, 5.7552E+04,
+	 true      , 1.0043E+00, 1.0853E+00, 2.5010E+06, 1.8000E+03, 2.9092E+02, 3.0417E-03, 7.0827E-05,-6.5924E+09,-7.0827E-05,
+	 -8.8982E+07},
+
+	{4.2614E-08, 1.8780E+07, 2.9903E-11, 1.3178E+04, 0.0000E+00, 8.3182E+03, 0.0000E+00, 0.0000E+00, 0.0000E+00, 5.6939E+04,
+	 true      , 1.0151E+00, 1.0911E+00, 2.5010E+06, 1.8000E+03, 2.9119E+02, 2.7224E-03, 7.6760E-05,-6.5494E+09,-7.6760E-05,
+	 -9.0523E+07},
+
+	{4.5927E-08, 1.8780E+07, 3.2867E-11, 1.3440E+04, 0.0000E+00, 8.4833E+03, 0.0000E+00, 0.0000E+00, 0.0000E+00, 5.6317E+04,
+	 true      , 1.0263E+00, 1.0970E+00, 2.5010E+06, 1.8000E+03, 2.9143E+02, 2.4202E-03, 8.2728E-05,-6.5044E+09,-8.2728E-05,
+	 -9.0778E+07},
+      };
+
+      // Sync to device
+      view_1d<P3UpdatePrognosticLiqData> pupldc_device("pupldc", Spack::n);
+      auto pupldc_host = Kokkos::create_mirror_view(pupldc_device);
+
+      // This copy only copies the input variables.
+      std::copy(&pupldc[0], &pupldc[0] + Spack::n, pupldc_host.data());
+      Kokkos::deep_copy(pupldc_device, pupldc_host);
+
+      // Get data from fortran
+      for (Int i = 0; i < max_pack_size; ++i) {
+	update_prognostic_liquid(pupldc[i]);
```

Comment:
Strange indenting here.

Reply:
This indenting and the indenting near lines 618 shows okay in emacs but not here on github....I am not sure why.

Reply:
I'm not sure how emacs and github handle tabs, but in vim tabs can be automatically expanded to spaces, and the number of spaces can be configured.

Perhaps you have tabs, which expand to the correct indentation on your emacs, but to something different in github?

Reply:
Thanks @bartgol . I changed the tab spacing to "spaces" and it seemed to work.